beacon-chain/rpc/eth/shared/structs.go

```diff
@@ -101,6 +101,32 @@ type VoluntaryExit struct {
 	ValidatorIndex string `json:"validator_index" validate:"required,number,gte=0"`
 }
 
+type Fork struct {
+	PreviousVersion string `json:"previous_version"`
+	CurrentVersion  string `json:"current_version"`
+	Epoch           string `json:"epoch"`
+}
+
+func (s *Fork) ToConsensus() (*eth.Fork, error) {
```

Comment:
This doesn't appear to be used anywhere.

Reply:
i'll be using it in the state api so i think it's fine to leave for now